modules/rte_mis_mail/src/Form/SMSSettingsForm.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Drupal\rte_mis_mail\Form;
+
+use Doctrine\Common\Annotations\Annotation\Required;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\mobile_number\MobileNumberUtilInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a form to configure Mobile Number settings.
+ */
+class SMSSettingsForm extends ConfigFormBase {
+
+  /**
+   * The Mobile Number Utility service.
+   *
+   * @var \Drupal\mobile_number\MobileNumberUtilInterface
+   */
+  protected $mobileNumberUtil;
+
+  /**
+   * Constructs a MobileNumberSettingsForm object.
+   *
+   * @param \Drupal\mobile_number\MobileNumberUtilInterface $mobileNumberUtil
+   *   The Mobile Number Utility service.
+   */
+  final public function __construct(MobileNumberUtilInterface $mobileNumberUtil) {
+    $this->mobileNumberUtil = $mobileNumberUtil;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('mobile_number.util')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'mobile_number_config_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['custom_sms_settings.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('custom_sms_settings.settings');
+
+    $form['heading'] = [
+      '#type' => 'markup',
+      '#markup' => '<h2>SMS Services</h2>',
+  ];
+    
+    $form['verify'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Mobile Number Verification'),
+      '#default_value' => $config->get('verify'),
+      '#description' => $this->t('Verification requirement.'),
+    ];
+    $form['message'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Verification Message'),
+      '#default_value' => $config->get('message'),
+      '#description' => $this->t('The SMS message to send during verification.'),
+      '#states'=>['visible' => [
+        ':input[name="verify"]' => 
+          ['checked' => TRUE],
+        ],
+      ]
+    ];
+    
+    return parent::buildForm($form, $form_state);
+  }
```

Comment:
Can we combine both Email & SMS settings

Reply:
Sure we can.